core/livepeernode.go

```diff
@@ -269,8 +278,7 @@ func (n *LivepeerNode) transcodeAndBroadcastSeg(seg *stream.HLSSegment, sig []by
 
 	transcodeStart := time.Now().UTC()
 	// Ensure length matches expectations. 4 second + 25% wiggle factor, 60fps
-	err := ffmpeg.CheckMediaLen(fname, 4*1.25*1000, 60*4*1.25)
-	if err != nil {
+	if err := ffmpeg.CheckMediaLen(fname, 4*1.25*1000, 60*4*1.25); err != nil {
```

Comment:
This was because golang was complaining about err being already initialized (from the `ReceiptExists` check above).